docker/ml-testing-toolkit/spec_files/user_config.json

```diff
@@ -1,6 +1,6 @@
 {
   "VERSION": 1,
-  "CALLBACK_ENDPOINT": "http://account-lookup-service:4002",
+  "CALLBACK_ENDPOINT": "http://example-users-api-service:8000",
```

Comment:
is this intentional?

Reply:
@shashi165 yes. This was the default or placeholder value, I had changed it to run the tests locally then unintentionally pushed the change in an earlier PR. This reverts that change.